src/blog/_posts/2023-06-08-survey-results.md

```diff
@@ -0,0 +1,54 @@
+---
+title: 2023 Cadence Community Survey Results
+
+date: 2023-06-08
+author: Ender Demirkaya
+authorlink: https://www.linkedin.com/in/enderdemirkaya/
+---
+
+We have released a user survey earlier this year to learn about who our users were, how they use Cadence and how we could help them. After collecting your great feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable to us and it helps us shape our roadmap for the future.
```

Comment:
We released a user survey earlier this year to learn about who our users are, how they use Cadence, and how we can help them. After collecting your great feedback, we wanted to share the results with our community. Thank you everyone for filling it out! Your feedback is invaluable and it helps us shape our roadmap for the future.